### DISCUSSION TITLE
Modify code for new device function

### DISCUSSION THREAD

**Discussion by Marvin-VW** [andrei-tatar/node-red-contrib-smartnora#295] [General]
I want to add more functions to the light. But when i clone your repo and modify the code what is the next step to get it running in node red? What do i have to do to import it in node-red after modifying the code?

**Comment by andrei-tatar** [comment]
@Marvin-VW I usually use `npm link` to run the plugin code (once compiled) together with node-red. I haven't tried getting a debugger attached (yet), not sure if node-red even loads the ts node files.
